content/en/synthetics/ci.md

```diff
@@ -192,11 +192,66 @@ yarn add --dev @datadog/datadog-ci
 {{% /tab %}}
 {{< /tabs >}}
 
-### Adding and running tests
+### Setup the integration
```

Comment:
Did you change to `integration` on purpose? That's a quite different beast than regular Datadog integration.

Reply:
Changing to client and resolving